handlers/nexmo/handler.go

```diff
@@ -227,15 +227,18 @@ func (h *handler) Send(ctx context.Context, msg courier.MsgOut, res *courier.Sen
 		}
 
 		nexmoStatus, err := jsonparser.GetString(respBody, "messages", "[0]", "status")
+		errCode, _ := strconv.Atoi(nexmoStatus)
 		if err != nil || nexmoStatus != "0" {
-			return courier.ErrResponseUnexpected
+			// https://developer.vonage.com/messaging/sms/guides/troubleshooting-sms
+			clog.Error(courier.ErrorExternal("send:"+nexmoStatus, sendErrorCodes[errCode]))
+			return courier.ErrFailedWithReason(fmt.Sprint(errCode), sendErrorCodes[errCode])
```

Comment:
we don't want to these both because the error will be converted to a channel log error

Reply:
the code here needs to be the way it was previously: `"send:"+nexmoStatus` to work with 

https://github.com/nyaruka/rapidpro/blob/832d6193a8bb9b5177bbe20d753f6ea2132d54fe/temba/channels/types/vonage/type.py#L117-L122

Reply:
the code here needs to be the way it was previously: "send:"+nexmoStatus to work with

https://github.com/nyaruka/rapidpro/blob/832d6193a8bb9b5177bbe20d753f6ea2132d54fe/temba/channels/types/vonage/type.py#L117-L122